block_iagora.php

```diff
@@ -61,4 +69,27 @@ public function get_content() {
     public function applicable_formats() {
         return ['all' => true];
     }
+     
+    /**  
+     * Allow the block to be configured.
+     * 
+    * @return bool
+    */
+    public function instance_allow_config() {
+        return true;
+    }
+
+    /**
+     * Save instance configuration.
+     * @param stdClass $data The configuration data.
+     * @param bool $nolongerused Indicates if $data is no longer used.
+     * @return bool True if data was saved successfully, false otherwise.
+     */
```

Comment:
```suggestion
    /**
     * Save instance configuration.
     *
     * @param stdClass $data The configuration data.
     * @param bool $nolongerused Indicates if $data is no longer used.
     * @return bool True if data was saved successfully, false otherwise.
     */
```